### PR TITLE
Fix site build.

### PR DESCRIPTION
**Pull request by karelmaxa**
I fixed build of the project site. The most significant changes:

* Fixed some unescaped HTML sequences in  Javadocs.
* Existing module site pages were rewritten (primarily was performed rebranding to Wren Security).
* Globally suppressed Javadoc linting for `maven-javadoc-plugin`.
* Some CCPL man pages were removed.

Maven `site` goal is now working and it is producing sites with relevant content.

### CODE REVIEW

File: opendj-doc-generated-ref/pom.xml [hunk 1]
```diff
@@ -185,21 +185,11 @@
...
                             <tools>
-                                <tool>
-                                    <name>addrate</name>
-                                    <application>com.forgerock.opendj.ldap.tools.AddRate</application>
-                                    <trailingSectionPaths>
-                                        <trailingSectionPath>exit-codes-0-80-89.xml</trailingSectionPath>
-                                        <trailingSectionPath>addrate-examples.xml</trailingSectionPath>
-                                    </trailingSectionPaths>
-                                </tool>
-
```
Comment (pavelhoral) [inline opendj-doc-generated-ref/pom.xml:196]: I would like to have some sort of comment in place of deleted CPPL content. E.g.

```xml
<!-- TODO add addrate documentation -->
```

And if it is not necessary, then do not remove the whole section... maybe just the CPPL content?

File: opendj-doc-generated-ref/src/main/assembly/sdk-tools-man-pages-assembly.xml [hunk 2]
```diff
@@ -27,11 +27,6 @@
     The .jar to has this file at /man-pages/man-addrate.xml for example.
   -->
   <files>
```
Comment (pavelhoral) [inline opendj-doc-generated-ref/src/main/assembly/sdk-tools-man-pages-assembly.xml:29]: Similar comment as in the previous case... by deleting these references we are loosing information about what we should add in the future. I would either comment these references out or replace content of the referenced files with some sort of `TODO` comment.

File: opendj-ldap-toolkit/src/main/java/com/forgerock/opendj/ldap/tools/AddRate.java [hunk 3]
```diff
@@ -435,7 +434,6 @@ int run(final String[] args) {
                     StringArgument.builder(MakeLDIF.OPTION_LONG_RESOURCE_PATH)
                             .shortIdentifier('r')
                             .description(INFO_ADDRATE_DESCRIPTION_RESOURCE_PATH.get())
-                            .docDescriptionSupplement(SUPPLEMENT_DESCRIPTION_RESOURCE_PATH.get())
```
Comment (pavelhoral) [inline opendj-ldap-toolkit/src/main/java/com/forgerock/opendj/ldap/tools/AddRate.java:438]: Is this change intentional?

File: opendj-ldap-toolkit/src/main/java/com/forgerock/opendj/ldap/tools/AuthRate.java [hunk 4]
```diff
@@ -381,7 +381,6 @@ int run(final String[] args) {
                 .build();
         argParser.setVersionHandler(newSdkVersionHandler());
         argParser.setShortToolDescription(REF_SHORT_DESC_AUTHRATE.get());
-        argParser.setDocToolDescriptionSupplement(SUPPLEMENT_DESCRIPTION_RATE_TOOLS.get());
```
Comment (pavelhoral) [inline opendj-ldap-toolkit/src/main/java/com/forgerock/opendj/ldap/tools/AuthRate.java:384]: Is this change intentional?

File: opendj-ldap-toolkit/src/main/java/com/forgerock/opendj/ldap/tools/MakeLDIF.java [hunk 5]
```diff
@@ -122,7 +121,6 @@ int run(final String... args) throws LDAPToolException {
                     StringArgument.builder(OPTION_LONG_RESOURCE_PATH)
                             .shortIdentifier('r')
                             .description(INFO_MAKELDIF_DESCRIPTION_RESOURCE_PATH.get())
-                            .docDescriptionSupplement(SUPPLEMENT_DESCRIPTION_RESOURCE_PATH.get())
```
Comment (pavelhoral) [inline opendj-ldap-toolkit/src/main/java/com/forgerock/opendj/ldap/tools/MakeLDIF.java:125]: Is this change intentional?

**Review by pavelhoral — CHANGES_REQUESTED** [review]
See my comments. I would like to have some sort of placeholder in place of the removed CCPL content if we would like to have that content replaced in the future.

File: opendj-doc-generated-ref/pom.xml (hunk 1 above)
Comment (karelmaxa) [reply opendj-doc-generated-ref/pom.xml]: I created empty man page files with CDDL license. So this change has been reverted.

File: opendj-doc-generated-ref/src/main/assembly/sdk-tools-man-pages-assembly.xml (hunk 2 above)
Comment (karelmaxa) [reply opendj-doc-generated-ref/src/main/assembly/sdk-tools-man-pages-assembly.xml]: I created empty man page files with CDDL license. So this change has been reverted.

File: opendj-ldap-toolkit/src/main/java/com/forgerock/opendj/ldap/tools/AddRate.java (hunk 3 above)
Comment (karelmaxa) [reply opendj-ldap-toolkit/src/main/java/com/forgerock/opendj/ldap/tools/AddRate.java]: The change was intentional because the source file was deleted. But it is not relevant anymore, source files were reverted back.

**Review by pavelhoral — APPROVED** [review]
LGTM
